packages/editor/src/components/page-attributes/parent.js

```diff
@@ -83,8 +82,8 @@ export function PageAttributesParent() {
 		[ fieldValue ]
 	);
 
-	const isHierarchical = get( postType, [ 'hierarchical' ], false );
```

Comment:
Here we could just return `isHiearchical` (and `parentPageLabel`, too) from the `useSelect` callback few lines above. It already computes it.

Reply:
Good idea, reused in bfdd8424e431add15e6300d600942a1173aaaf69.